doc/ref_cert/README.md

```diff
@@ -1,5 +1,49 @@
 [<< Back](../)
 
+# CNTT Reference Conformance (RC) Requirements and Testing Principles
+
+The objective each RC workstream is to provide an automated mechanism to measure either a network function (NF) or a cloud infrastructure with a standard set of requirements defined by the associated Reference Architecture (RA).  Through this measurement, a provider of either NFs or cloud infrastructure will be able to test their conformance to the RA.  This will ease the integration of NFs into operator environments that host compatible cloud infrastructures, thereby reducing cost, complexity, and time of integration.
+
+The overall program requires the close coordination of the following:
```

Comment:
Program seems induced by a copy/paste from OVP. I would consider it as irrelevant here.

Reply:
How is it irrelevant? It is describing what the RC does. It has nothing to do with OVP